CONTRIBUTING.md

```diff
@@ -1,39 +1,167 @@
+![Commit Strip
+http://www.commitstrip.com/en/2014/05/07/the-truth-behind-open-source-apps/](docs/how-open-source-maintained.jpg)
+
+## Common issues and resolutions
+
+- Using `grape-active_model_serializers`, or any non-Rails server. See
+    [issue](https://github.com/rails-api/active_model_serializers/issues/1258).
+
 ## How can I help?
 
-Everyone is encouraged to open issues that are affecting you: bugs, ideas, performance problems – everything helps!
+### Filing an issue
+
+Everyone is encouraged to open issues that are affecting you:
```

Comment:
but I'm going to write `Everyone is encouraged to open issues that are affecting them:`